modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/taglib/_vertical_card.scss

```diff
@@ -49,3 +49,12 @@
 .taglib-vertical-card-footer {
 	min-height: 5.6em;
 }
+
+.rtl {
+	.card .aspect-ratio-item-center-middle {
+		left: auto;
+		margin-right: 1.5rem;
+		right: 50%;
+		transform: translate(-100%, -50%);
+	}
+}
```

Comment:
We pretty rarely have to explicitly target `.rtl` (I count 16 hits in all of liferay-portal). Any idea why the transform that our frontend-css-rtl-servlet does isn't covering us here? cc @eduardoallegrini

Reply:
The script is working but it's not changing the `translate` values

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/46778114/90411972-cf970880-e0ac-11ea-94b0-d8ebdc909171.gif)

So
```scss
.rtl {
	.card .aspect-ratio-item-center-middle {
		transform: translate(50%, -50%);
	}
}
```
Should be enough